eng/SignCheckExclusionsFile.txt

```diff
@@ -13,3 +13,4 @@
 *comhosttemplatecomhostdll.dll;;Template, DO-NOT-SIGN, https://github.com/dotnet/core-setup/pull/7549
 *staticapphosttemplateapphostexe.exe;;Template, DO-NOT-SIGN, https://github.com/dotnet/core-setup/pull/7549
 *dotnet.js;;Workaround, https://github.com/dotnet/core-eng/issues/9933
+*dotnet_crypto_worker.js;;Workaround, https://github.com/dotnet/core-eng/issues/9933
```

Comment:
@ViktorHofer just curious, what is the purpose of this file? From the linked issue, it seems that signing validation happens in an outer ring - in which case what is the purpose of these exclusions. Just curious

Reply:
That file predates the outer ring signing validation. @mmitche do you know if the Validate-DotNet outer ring validation honors the SignCheckExclusionFile in our repo?

Reply:
Also just saw that the outer ring signing validation is failing and opened https://github.com/dotnet/core-eng/issues/12601 to better understand who tracks that pipeline.